### DISCUSSION TITLE
Option for chaos client to get all subdomains from https://dns.projectdiscovery.io/dns/

### DISCUSSION THREAD

**Discussion by 0xtavi** [projectdiscovery/chaos-client#46] [Ideas]
It would be nice to have an option like `chaos all` which returns all the subdomains stored
Btw is this option available on https://dns.projectdiscovery.io/dns/ somehow?

**Comment by ehsandeep** [comment]
Not sure you noticed, you can do this directly from website @0xtavi

![image](https://user-images.githubusercontent.com/8293321/113485244-3517f800-94ca-11eb-9181-385c28d62112.png)

**Reply by MetzinAround** [reply]
Closing as this answers. Thanks!
